docs/resources/widgets/pages/pages.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Pages
```

Comment:
Same comment as widgets, I think we need a file nested under the pages section called "Whats a page?" or something. It wasn't clear to me that clicking on Page would bring me to an actual page

Reply:
Yep, corrected!